stock_picking_quick/models/product_product.py

```diff
@@ -0,0 +1,53 @@
+# © 2022 Today Akretion
+# @author Pierrick Brun <pierrick.brun@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from odoo import api, fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    move_ids = fields.One2many(
```

Comment:
There is already that field in core : 

https://github.com/odoo/odoo/blob/14.0/addons/stock/models/product.py#L27